requirements-hyperopt.txt

```diff
@@ -7,3 +7,4 @@ scikit-learn==0.22.2.post1
 scikit-optimize==0.7.4
 filelock==3.0.12
 joblib==0.14.1
+tqdm==4.43.0
```

Comment:
Please make sure to add this to setup.py as well ... 

installation from pypi (`pip install freqtrade[all]` doesn't use the requirements.txt files, but relies on what's specified in setup.py).

Reply:
I always forgetting this... 👍

Reply:
Good point will do with next commit

Reply:
i see a potential problem with the implementation in setup.py.. doesn't seem to have version control so it will install newer compared to requirements files if it exists = not tested.

Reply:
can't be "fixed"